idl/github.com/uber/cadence/shared.thrift

```diff
@@ -1468,6 +1468,24 @@ struct DescribeTaskListResponse {
   20: optional TaskListStatus taskListStatus
 }
 
+struct GetTaskListPartitionInfoResponse {
+  10: optional PartitionInfo activityTaskPartition
+  20: optional PartitionInfo decisionTaskPartition
+}
+
+struct PartitionInfo {
```

Comment:
Do you really need this struct ? Why not declare the map<> inside your Response struct ? If you do want to keep this struct, I would suggest not calling it PartitionInfo - because the struct contains info about all partitions not just one.